packages/mobile/src/components/details-tile/DetailsTileHasAccess.tsx

```diff
@@ -30,22 +31,33 @@ const messages = {
   specialAccess: 'SPECIAL ACCESS',
   payToUnlock: 'Pay to Unlock',
   unlockedCollectibleGatedPrefix: 'A Collectible from ',
-  unlockedCollectibleGatedSuffix:
-    ' was found in a linked wallet. This track is now available.',
+  unlockedCollectibleGatedSuffix: (
```

Comment:
we don't actually need these right now right? you're just adding in preparation for future?

Reply:
Yeah correct, I was actually adding a comment to explain it. Technically extra but iImo, there should be no harm adding it in this component since `contentType` is already available.